articles/aks/deploy-spinkube.md

```diff
@@ -0,0 +1,362 @@
+  --create-namespace \
+  --version 0.2.3 \
+  --set kwasmOperator.installerImage=ghcr.io/spinkube/containerd-shim-spin/node-installer:v0.15.1
+```
+
+#### Provision containerd-shim-spin to Kubernetes nodes
+
+Once `runtime-class-manager` is installed on your AKS cluster, you must annotate the Kubernetes nodes that should be able to run Spin Apps with `kwasm.sh/kwasm-node=true`. You can use `kubectl annotate node` to annotate either a subset of the nodes available in your AKS cluster or - as shown in the following snippet - all nodes of your AKS cluster:
+
+```azurecli-interactive
+# Provision containerd-shim-spin to all nodes
+kubectl annotate node --all kwasm.sh/kwasm-node=true
+```
+
+After annotating the Kubernetes node(s), `runtime-class-manager` will use a Kubernetes *Job* to modify the desired node(s). After successful deployment of `containerd-shim-spin`, the node(s) will be labeled with a `kwasm.sh/kwasm-provisioned` label. You can check if the desired node(s) have the `kwasm.sh/kwasm-provisioned` label assigned using the `kubectl get nodes --show-labels` command:
+
+```azurecli-interactive
+# Verify kwasm.sh/kwasm-provisioned is set on desired node(s)
+kubectl get nodes --show-labels
+```
+
+### Deploy the `spin-operator`
+
+The `spin-operator` consists of two Custom Resource Definitions (CRDs), the RuntimeClass for `spin`, and a `SpinAppExecutor`, which must be deployed to your AKS cluster. 
+
+Start by deploying the CRDs and the RuntimeClass for `spin`:
+
+```azurecli-interactive
+kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.2.0/spin-operator.crds.yaml
+kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.2.0/spin-operator.runtime-class.yaml
+```
+
+The `spin-operator` itself is deployed using a Helm chart:
+
+```azurecli-interactive
+helm install spin-operator --version 0.2.0 \
+  --namespace spin-operator --create-namespace \
+  --wait oci://ghcr.io/spinkube/charts/spin-operator
+```
+
+Finally, create a `SpinAppExecutor` in the default namespace:
+
+```azurecli-interactive
+kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.2.0/spin-operator.shim-executor.yaml
+```
+
+## Run a Spin App on AKS
+
+In this section you will verify the SpinKube installation, by creating a simple Spin App using the `spin` CLI and JavaScript. Start by creating a new Spin App using the `http-js` template:
+
+```azurecli-interactive
+# Create a new Spin App
+spin new -t http-js --accept-defaults hello-spinkube
+
+# Move into the application folder
+cd hello-spinkube
+
+# Install Wasm Tooling
+npm install
+```
+
+Use the `spin` CLI create a basic *Hello, World* application:
+
+```azurecli-interactive
+# Compile the app to Wasm
+spin build
+```
+
+Spin Apps are packaged as OCI artifacts and distributed via OCI compliant artifacts such as Azure Container Registry (ACR). Create a new ACR instance and grab necessary credentials for authenticating the `spin` CLI:
```

Comment:
```suggestion
Spin Apps are packaged as OCI images and distributed via an OCI compatible registry such as Azure Container Registry (ACR). Create a new ACR instance and grab necessary credentials for authenticating the `spin` CLI:
```

Reply:
I prefer to stick with OCI artifacts instead of OCI images when talking about Spin Apps, to reduce confusion. Obviously, the other findings will be addressed